dm/ctl/common/util.go

```diff
@@ -221,3 +221,22 @@ func IsDDL(sql string) (bool, error) {
 		return false, nil
 	}
 }
+
+// GetNameFromArgOrFile tries to retrieve name field from the file if arg is filename-like, otherwise returns arg directly
+func GetNameFromArgOrFile(arg string) string {
+	if !strings.HasSuffix(arg, ".yaml") {
+		return arg
+	}
+	var (
+		content []byte
+		err     error
+	)
+	if content, err = GetFileContent(arg); err != nil {
+		return arg
+	}
+	cfg := config.NewTaskConfig()
```

Comment:
This function only handles task names, so how about renaming it to something like `GetTaskNameFromArgOrFile`?

Reply:
https://github.com/pingcap/dm/pull/854/commits/7b5548113e45700a8e24536431d9c9aecac9fe83